staff/adrian-zhu/skylab-semantics/community.html

```diff
@@ -0,0 +1,198 @@
+<head>
+    <script></script>
+    <script></script>
+    <script></script>
+    <script></script>
+    <script></script>
+    <script></script>
+    <script></script>
+    <script></script>
+    <meta>
```

Comment:
is `meta` semantic?

Reply:
based on the information of https://developer.mozilla.org/en-US/docs/Web/HTML/Element, it actually shows that the 'meta' is semantic element